core/shared/src/main/scala/fs2/async/mutable/Queue.scala

```diff
@@ -144,10 +160,14 @@ object Queue {
       * @param queue    Queue, expressed as vector for fast cons/uncons from head/tail
       * @param deq      A list of waiting dequeuers, added to when queue is empty
       */
-    final case class State(queue: Vector[A], deq: Vector[Ref[F,Chunk[A]]])
+    final case class State(
+      queue: Vector[A],
+      deq: Vector[Ref[F,Chunk[A]]],
+      peek: Option[Ref[F, A]]
```

Comment:
Adding a `@param` for this as well would be nice, for consistency

Reply:
Ok.